haystack/utils/doc_store.py

```diff
@@ -103,10 +102,6 @@ def stop_elasticsearch(delete_container=False):
     stop_container(ELASTICSEARCH_CONTAINER_NAME, delete_container)
 
 
-def stop_milvus(delete_container=False):
-    stop_container(MILVUS1_CONTAINER_NAME, delete_container)
-
-
```

Comment:
Should we rather make this function stop the Milvus2 containers? I don't think it's being used much, it's more about consistency, as the other docstores all have such function

Reply:
We don't have a corresponding "launch milvus2" function, and I think we shouldn't embark in supporting Docker orchestration of Milvus 2.0... If anything, I would rather remove this whole module to bring in consistency :)